recipes/sparse-dot-topn/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "sparse_dot_topn" %}
+{% set version = "0.2.5" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "2028d451d81cf5a73cf9d5fc56de35f80ac8dbd528a6abe3b723bec9c1adcc38"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - pip
+    - python
+    - numpy
+    - cython
+    - setuptools >=18.0
+  run:
+    - python
+    - numpy
```

Comment:
Also, please follow this pattern if building against numpy: https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=numpy#building-against-numpy

Reply:
@scopatz got it - fixed below. thanks!